python/ray/data/tests/test_file_based_datasource.py

```diff
@@ -9,22 +8,6 @@
 from ray.data._internal.delegating_block_builder import DelegatingBlockBuilder
 from ray.data.block import Block
 from ray.data.datasource.file_based_datasource import FileBasedDatasource
-from ray.data.datasource.path_util import _has_file_extension, _is_local_windows_path
-
-
-@pytest.mark.parametrize(
-    "path, extensions, has_extension",
-    [
-        ("foo.csv", ["csv"], True),
-        ("foo.csv", ["json", "csv"], True),
-        ("foo.csv", ["json", "jsonl"], False),
-        ("foo.parquet.crc", ["parquet"], False),
-        ("foo.parquet.crc", ["crc"], True),
-        ("foo.csv", None, True),
-    ],
-)
-def test_has_file_extension(path, extensions, has_extension):
-    assert _has_file_extension(path, extensions) == has_extension
 
```

Comment:
I moved the other `path_util` tests out of `test_file_based_datasource` to a separate `test_path_util` file